File: public/docs/_examples/dependency-injection/ts/app/app.config.ts
//#docregion
// #docregion token
import {OpaqueToken} from 'angular2/core';

export let APP_CONFIG = new OpaqueToken('app.config');
// #enddocregion token

//#docregion config
export interface Config {
  apiEndpoint: string,
  title: string
}

export const CONFIG:Config = {
  apiEndpoint: 'api.heroes.com',
  title: 'Dependency Injection'
};
//#enddocregion config

File: public/docs/_examples/dependency-injection/dart/bin/app.config.dart
//#docregion

// #docregion token
import "package:angular2/core.dart";

const APP_CONFIG = const OpaqueToken("app.config");
// #enddocregion token

//#docregion config
class Config {
  final String apiEndpoint;
  final String title;

  const Config({this.apiEndpoint, this.title});
}

const Config CONFIG =  const Config(
  apiEndpoint: "api.heroes.com",
  title: "Dependency Injection"
);
//#enddocregion config
